src/commands.ts

```diff
@@ -476,6 +478,39 @@ async function deployFunction(
   }
 }
 
+export async function deleteProjectHandler(projectId : string, forced : boolean): Promise<boolean> {
+  // show prompt if no project id is selected
+  if (typeof projectId === 'string' && projectId.trim().length === 0) {
+    console.log('No project ID specified, select an ID to delete from this list:')
+    const projects = await listProjects();
+    if (projects.length === 0) {
```

Comment:
It would have shown "No project ID specified, select an ID to delete from this list:" and then "There are no currently deployed projects". I've changed the order of the checks and messages accordingly.